app/src/main/res/menu/menu_main.xml

```diff
@@ -1,14 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file could be removed as it is no longer required. and as the references of id have been renamed to R.id.info.